src/drop-down-icon.jsx

```diff
@@ -112,7 +112,7 @@ const DropDownIcon = React.createClass({
           <div onTouchTap={this._onControlClick}>
               <FontIcon
                 className={iconClassName}
-                style={this.prepareStyles(iconStyle)}>{this.props.iconLigature}</FontIcon>
+                style={iconStyle}>{this.props.iconLigature}</FontIcon>
```

Comment:
I don't see why you can't. `iconStyle` is still passed down to the internal `<div>` generated by `<FontIcon>`